dao/src/main/java/uk/gov/moj/sdt/dao/IndividualRequestDao.java

```diff
@@ -169,18 +169,17 @@ public List<IIndividualRequest> getStaleIndividualRequests(final int minimumAgeI
         final LocalDateTime now = LocalDateTime.now();
         final LocalDateTime latestTime = now.minusMinutes(minimumAgeInMinutes);
         List<IndividualRequest> queryList = this.queryAsList(IndividualRequest.class, () -> {
-            Predicate[] sdtCustomerPredicate =   createIndividualRequestPredicate(false, latestTime);
+            Predicate[] sdtCustomerPredicate =   createIndividualRequestPredicate(latestTime);
             return criteriaQuery.select(root).where(sdtCustomerPredicate);
         });
         return new ArrayList<>(queryList);
     }
 
-    private Predicate[] createIndividualRequestPredicate(boolean deadLetter, LocalDateTime latestTime) {
+    private Predicate[] createIndividualRequestPredicate(LocalDateTime latestTime) {
```

Comment:
I can see the reasoning behind removing the deadLetter parameter - the original script set the deadLetter flag to false for all stuck requests before requeuing them, so removing it from here is sort of equivalent.  The only issue is that if the deadLetter flag isn't set to false we could have requests that are successfully sent by the requeue, but still have the deadLetter flag set to true.  This means the deadLetter flag may not be accurate.

Check and see if the deadLetter flag is set to false if the request is successfully processed.  If it isn't, we'll have to include a step in `RequeueOldIndividualRequests` that sets the flag to false before calling the requeue method.

Reply:
done now

Reply:
`SdtManagementMBean.requeueOldIndividualRequests()` changed to set deadLetter flag to false when iterating through records returned by this method.